web/public/config.js

```diff
@@ -7,7 +7,7 @@
 
 var config = {
   base_url: window.location.origin, // Change to test against a different server
-  app_root: "/app",
+  app_root: "/",
```

Comment:
Good change. I wonder how long it took you to figure this one out. I even stumble over this occasionally. "WHY IS THE '/' PAGE BLANK???"